ludwig/config_validation/validation.py

```diff
@@ -28,11 +28,19 @@ def get_schema(model_type: str = MODEL_ECD):
 
     cls = model_type_schema_registry[model_type]
     props = unload_jsonschema_from_marshmallow_class(cls)["properties"]
+
+    # TODO: Replace with more robust required logic later.
+    required = []
+    if model_type == MODEL_ECD or model_type == MODEL_GBM:
+        required = ["input_features", "output_features"]
```

Comment:
This is actually true for all model types including LLMs

Reply:
good catch